### ISSUE TITLE
Can't import 'FirebaseAuthState'.

### ISSUE THREAD

**Issue by YogliB** [angular/angularfire2#959]
I get the following error, trying to import:
"Module '".../node_modules/angularfire2/auth"' has no exported member 'FirebaseAuthState'."

And this one, trying a different path:
Module '".../node_modules/angularfire2/index"' has no exported member 'FirebaseAuthState'.

EDIT: I'm trying to use this function from the docs:
```
  signInWithFacebook(): firebase.Promise<FirebaseAuthState> {
    return this.afAuth.auth.signInWithPopup(new firebase.auth.FacebookAuthProvider());
  }
```
(https://github.com/angular/angularfire2/blob/master/docs/Auth-with-Ionic2.md)

I use the latest version (4.0.0-rc.0), in an Ionic app (I'm using the guide in the docs).

**Comment by markterrill** [comment]
Hi,  using 4.0 RC0?
Try:
`private authState: Observable<firebase.User>;`
ie, not firebaseauthstate

you should `import * as firebase from 'firebase/app';` above as well.

this doc is the best if you happen to be using RC0: https://github.com/angular/angularfire2/blob/4.0.0-rc.0/docs/version-4-upgrade.md

**Comment by markgoho** [comment]
Hi @YogliB this is addressed in #965

**Comment by mukesh51** [comment]
@YogliB Try this to create an[ Ionic App3 with Angular4](https://github.com/angular/angularfire2/blob/master/docs/Auth-with-Ionic3-Angular4.md) from scratch.
